src/App.tsx

```diff
@@ -8,11 +8,52 @@ const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+type State = {
+  selectedTab: Tab | null;
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    selectedTab: tabs[0],
+  };
+
+  onTabSelected = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { id } = event.target;
+
+    const currentTab = tabs.find(tab => tab.id === id) || null;
```

Comment:
just do like this, so now getTabId method useless 
```suggestion
    const currentTab = tabs.find(tab => tab.id === id) || tabs[0];
```